### ISSUE TITLE
[Feature request]: Migrate data

### ISSUE THREAD

**Issue by DonKoko** [Shelf-nu/shelf.nu#724]
### Is your feature request related to a problem? Please describe?

Lots of users start on a personal workspace but want to switch to team. Currently its possible but there are quite some limitations and issues related to uniqueness of QR codes, so its not a real migration and QR codes will never be migrated as we are creating new assets. 

### Describe the solution you'd like

Migrate data feature.
This feature will allow you to choose a target workspace/organization and will move all related entries from `currentOrganization` to `targetOrganization`.

**Comment by carlosvirreira** [comment]
Our latest experiment and solution kind solves this challenge right @DonKoko ?

**Comment by DonKoko** [comment]
Yes, but still needs to be implemented code wise. Also the solution we made has some issues in terms what the user might want, however I have some other ideas. 

So the general concept is that instead of moving assets around and updated hundreds if not thousands of entries in the DB, we can instead just edit the actual workspaces by changing around their type and name, however this means we are swapping around so entries from workspace A to got workspace B and the other way around. If we dont want to swap but have it feel like they moved, we might need to do some other magic like creating a new empty workspace and doing some more renaming and changing of workspace type in the background.
Once I get to it I will outline the whole code strategy for this.

**Comment by carlosvirreira** [comment]
> Yes, but still needs to be implemented code wise. Also the solution we made has some issues in terms what the user might want, however I have some other ideas.
> 
> So the general concept is that instead of moving assets around and updated hundreds if not thousands of entries in the DB, we can instead just edit the actual workspaces by changing around their type and name, however this means we are swapping around so entries from workspace A to got workspace B and the other way around. If we dont want to swap but have it feel like they moved, we might need to do some other magic like creating a new empty workspace and doing some more renaming and changing of workspace type in the background. Once I get to it I will outline the whole code strategy for this.

Sounds great lets do it
